@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -215,14 +265,31 @@ export class SolidLineMaterial extends THREE.RawShaderMaterial
             defines.USE_DISPLACEMENTMAP = "";
         }
 
+        if (params !== undefined && !Boolean(params.dashColor)) {
+            defines.USE_DASH_ALPHA = "";
```

Comment:
You can use `updateDashAlpha` here

Reply:
@okanishchev for what reason? One more function call w/o actual benefit

Reply:
Here you have special semantics on enabling defines in shader code:
 * you need to use special name `USE_DASH_ALPHA`
 * you need to use special value `""`

And none of it is checked by typescript nor tslint. What effect it will have if I'll write for example
`defines.USE_DASHED_ALPHA = "";` or even
`defines.USE_DASH_ALPHA = 0;`

So using pre-defined function is less error-prone here.

Reply:
@okanishchev I don't see any use case where it would save the developer as it is an internal code. We force a consumer to use `updateDashAlpha()`, but this case is internal optimization and is unrelated. 
Plus it would break the consistency as USE_FOG is written the same way

Reply:
The internal code doesn't mean that you'll do better on understanding and maintaining the code. Here you can reuse the code you already have. But maybe in this particular case it's really doesn't matter.